packages/quantic/force-app/solutionExamples/main/lwc/exampleSearch/e2e/exampleSearch.e2e.ts

```diff
@@ -0,0 +1,151 @@
+      expect(searchPage.facetsManager).toBeVisible();
+      expect(searchPage.facets).toBeVisible();
+      expect(searchPage.summary).toBeVisible();
+      expect(searchPage.sort).toBeVisible();
+      expect(searchPage.tabBar).toBeVisible();
+      expect(searchPage.pager).toBeVisible();
+      expect(searchPage.foldedResultList).toBeVisible();
+      expect(searchPage.recentQueriesList).toBeVisible();
+      expect(searchPage.recentResultsList).toBeVisible();
+
+      const firstResult = searchPage.results.first();
+      expect(firstResult).toBeVisible();
+      expect(await firstResult.textContent()).not.toBeNull();
+
+      const resultsCount = searchPage.results.count();
+      expect(await resultsCount).toBeGreaterThan(0);
+    });
+  });
+
+  test.describe('the happy path', () => {
+    test('should allow the user to search, sort, change the result page, change the tab, select a facet and clear filters', async ({
+      searchPage,
+      search,
+    }) => {
+      // Trigger a search query
+      const exampleQuery = 'test';
+      const searchRequestPromise = search.waitForSearchRequest();
+
+      await searchPage.searchboxInput.fill(exampleQuery);
+      await searchPage.searchboxInput.press('Enter');
+
+      await searchRequestPromise;
+      const searchRequestBody = (await searchRequestPromise).postDataJSON();
+      expect(searchRequestBody?.q).toBe(exampleQuery);
+
+      await expect(searchPage.summary).toContainText(exampleQuery);
+
+      const recentQueriesListItems =
+        await searchPage.recentQueriesListItems.all();
+      expect(recentQueriesListItems.length).toEqual(1);
+      const expectedQueries = [exampleQuery];
+      expectedQueries.forEach(async (expectedQuery: string, index: number) => {
+        // eslint-disable-next-line no-await-in-loop
+        await expect(recentQueriesListItems[index]).toHaveText(expectedQuery);
+      });
+
+      // Change the sort order
+      const sortClickSearchRequestPromise = search.waitForSearchRequest();
+      await searchPage.sortDropdownButton.click();
+      const sortLabelWithValue = await searchPage.getSortLabelValue();
+      const {label: expectedSortName, value: expectedSortValue} =
+        sortLabelWithValue[1];
+
+      await searchPage.clickSortButton(expectedSortName);
+      const searchRequestBodyAfterSortClick = (
+        await sortClickSearchRequestPromise
+      ).postDataJSON();
+      expect(searchRequestBodyAfterSortClick?.sortCriteria).toBe(
+        expectedSortValue
+      );
+
+      // Change the result page
+      const pagerClickSearchRequestPromise = search.waitForSearchRequest();
+      await searchPage.clickOnPagerButtonByIndex(2);
+
+      const searchRequestBodyAfterPagerClick = (
+        await pagerClickSearchRequestPromise
+      ).postDataJSON();
+
+      expect(searchRequestBodyAfterPagerClick?.firstResult).toBe(10);
+
+      // Change the tab
+      const expectedTabName = 'Articles';
```

Comment:
This I would go actually get the text of the 2nd tab, so we're not dependant on our exampleSearch HTML for this test for example if we remove the Articles tab, or it's no longer the 2nd tab.